services/bettersocial/post/commentChild.js

```diff
@@ -151,13 +151,14 @@ const BetterSocialCreateCommentChildV3 = async (req) => {
       await countProcess(reaction_id, {comment_count: +1}, {comment_count: 1});
     }
 
-    await sendMultiDeviceReplyCommentNotification(
-      signedUserIdFeed,
-      req.user,
-      message,
-      reaction_id,
-      postTitle
-    );
+    if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId)))
+      await sendMultiDeviceReplyCommentNotification(
+        signedUserIdFeed,
+        req.user,
+        message,
+        reaction_id,
+        postTitle
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic to check if the user who made the comment is the same as the user who made the post before sending a notification seems correct. However, it would be better to handle potential exceptions that might occur during the execution of `UsersFunction.checkIsMe` and `sendMultiDeviceReplyCommentNotification`. This will ensure that even if there's an error in these functions, the rest of the code can still execute.
```diff
-    if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId)))
-      await sendMultiDeviceReplyCommentNotification(
-        signedUserIdFeed,
-        req.user,
-        message,
-        reaction_id,
-        postTitle
-      );
+    try {
+      if (!(await UsersFunction.checkIsMe(User, signedUserIdFeed, userId))) {
+        await sendMultiDeviceReplyCommentNotification(
+          signedUserIdFeed,
+          req.user,
+          message,
+          reaction_id,
+          postTitle
+        );
+      }
+    } catch (error) {
+      console.error('Error while checking user or sending notification:', error);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->